docs/src/operations/setting-up-a-cluster.md

```diff
@@ -1,3 +1,172 @@
 # Setting up a Zeebe Cluster
 
-Coming Soon
+To setup a cluster you need to adjust the `cluster` section
+in the Zeebe configuration file. Below is a snipped
+of the default Zeebe configuration file, it should be self explanatory.
+
+```toml
+[cluster]
+
+# This section contains all cluster related configurations, to setup an zeebe cluster
+
+# Specifies the unique id of this broker node in a cluster.
+# The id should be between 0 and number of nodes in the cluster (exclusive).
+#
+# This setting can also be overridden using the environment variable ZEEBE_NODE_ID.
+# nodeId = 0
+
+# Controls the number of partitions, which should exist in the cluster.
+#
+# This can also be overridden using the environment variable ZEEBE_PARTITIONS_COUNT.
+# partitionsCount = 1
+
+# Controls the replication factor, which defines the count of replicas per partition.
+# The replication factor cannot be greater than the number of nodes in the cluster.
+#
+# This can also be overridden using the environment variable ZEEBE_REPLICATION_FACTOR.
+# replicationFactor = 1
+
+# Specifies the zeebe cluster size. This value is used to determine which broker
+# is responsible for which partition.
+#
+# This can also be overridden using the environment variable ZEEBE_CLUSTER_SIZE.
+# clusterSize = 1
+
+# Allows to specify a list of known other nodes to connect to on startup
+# The contact points of the management api must be specified.
+# The format is [HOST:PORT]
+# Example:
+# initialContactPoints = [ "192.168.1.22:26502", "192.168.1.32:26502" ]
+#
+# This setting can also be overridden using the environment variable ZEEBE_CONTACT_POINTS
+# specifying a comma-separated list of contact points.
+#
+# Default is empty list:
+# initialContactPoints = []
+```
+
+# Example
+
+In this example we will setup an Zeebe cluster with
+five brokers. Each broker needs to get an unique node id.
+To scale well, we will bootstrap five partitions
+with an replication factor of three. For more information about this,
+please take a look into the [Clustering](/basics/clustering.md) section.
```

Comment:
This link is dead for me locally, where should it point to?

Reply:
http://localhost:3000/basics/clustering.html